snap/snapcraft.yaml

```diff
@@ -0,0 +1,52 @@
+name: getting-things-gnome
+base: core20
+compression: lzo
+grade: stable
+license: GPL-3.0
+adopt-info: gtg
+confinement: strict
+
+apps:
+  gtg:
+    command: usr/bin/gtg
+    extensions: [gnome-3-38]
+    environment:
+      PYTHONPATH: $SNAP/usr/lib/python3/dist-packages:$PYTHONPATH
+    common-id: org.gnome.GTG
+    plugs: [network]
+
```

Comment:
```suggestion
slots:
    session-dbus-interface:
        interface: dbus
        name: org.gnome.GTG
        bus: session

```

Reply:
I got it... Because the integration with Hamster is made using D-BUS...